functions/dataManager/collectionManager.ts

```diff
@@ -0,0 +1,119 @@
+import { Game } from "../models/game";
+import { createGame, modifyGame, deleteGame, getGame } from "./gameManager";
+import { Collection } from "../models/collection";
+import { createGamePriceData, deleteGamePriceData, modifyGamePriceData } from "./gamePriceDataManager";
+import * as Config from "../shared/config/config"
+import * as Common from "../shared/common/collection";
+import { CollectionError } from "../error/collectionErrorHandler";
+import * as Interfaces from "../shared/interfaces/interfaces";
+let paginatedData = [] as any;
+
+export async function getCollection(collection: Collection) : Promise<[Game]> {
+  let params = {
+    TableName: Config.table,
+    KeyConditionExpression: `#partitionKey = :partitionKey AND begins_with(sortKey, :sortKey)`,
+    ExpressionAttributeNames: {
+        "#partitionKey": "partitionKey",
+    },
+    ExpressionAttributeValues: {
+        ":partitionKey": collection.partitionKey,
+        ":sortKey": `[CollectionItem]#[${collection.collectionType}]#[GameItem]`
+    }
+  };
+  
+  try {
+    paginatedData = [];
+    await getPaginatedData(params);     
+    let gameList = [] as any
+    if (paginatedData.length > 0) {
+      for (let game of paginatedData) {
+        let returnedGame = await Common.deserializeDynamoCollection(game);
+        gameList.push(returnedGame);
+      };     
+    }
+    return gameList; 
+  } catch (err: any) {
+    throw err;
+  }
+}
+
+export async function getAllCollectionItems(collectionType: string) {
+  let params = {
+    TableName: Config.table,
+    IndexName: "itemTypeIndex",
+    KeyConditionExpression: "#itemType = :itemType",
+    ExpressionAttributeNames: {
+      "#itemType": "itemType"
+    },
+    ExpressionAttributeValues: {
+      ":itemType": `[CollectionItem]#[${collectionType}]#[GameItem]`
+    }
+  };
+  
+  try {
+    paginatedData = [];
+    await getPaginatedData(params);
+    if (paginatedData.length > 0) {
+      for (let game of paginatedData) {
+        let returnedGame = await Common.monitorRunningPrice(game);
+        await Common.sendRunningPriceNotification(returnedGame);
```

Comment:
Weren't we only supposed to be sending the notification IF it crossed the desiredPrice threshold?